rocAL/rocAL_pybind/amd/rocal/plugin/tf.py

```diff
@@ -64,125 +70,106 @@ def __init__(self, pipeline, tensor_layout = types.NCHW, reverse_channels = Fals
         self.device_id = device_id
         self.reverse_channels = reverse_channels
         self.tensor_dtype = tensor_dtype
-        self.w = b.getOutputWidth(self.loader._handle)
-        self.h = b.getOutputHeight(self.loader._handle)
-        self.n = b.getOutputImageCount(self.loader._handle)
         self.bs = pipeline._batch_size
+        self.output_list = self.dimensions = self.dtype = None
         if self.loader._name is None:
             self.loader._name = self.loader._reader
-        color_format = b.getOutputColorFormat(self.loader._handle)
-        self.p = (1 if (color_format == int(types.GRAY)) else 3)
-        if self.tensor_dtype == types.FLOAT:
-            data_type="float32"
-        elif self.tensor_dtype == types.FLOAT16:
-            data_type="float16"
-        
-        if(types.NHWC == self.tensor_format):
-            if self.device == "cpu":
-                self.out = np.zeros((self.bs*self.n, int(self.h/self.bs), self.w, self.p), dtype = data_type)
-            else:
-                with cp.cuda.Device(device=self.device_id):
-                    self.out = cp.zeros((self.bs*self.n, int(self.h/self.bs), self.w, self.p), dtype = data_type)
-        else:
-            if self.device == "cpu":
-                self.out = np.zeros((self.bs*self.n, self.p, int(self.h/self.bs), self.w), dtype = data_type)
-            else:
-                with cp.cuda.Device(device=self.device_id):
-                    self.out = cp.zeros((self.bs*self.n, self.p, int(self.h/self.bs), self.w), dtype = data_type)
-                
+    
     def next(self):
         return self.__next__()
 
     def __next__(self):
-        if(b.isEmpty(self.loader._handle)):
-            timing_info = b.getTimingInfo(self.loader._handle)
-            print("Load     time ::",timing_info.load_time)
```

Comment:
Are we printing these data internally somewhere?

Reply:
No

Reply:
Then why have we removed it?